packages/wallet-manager/src/wallets/Substrate/Substrate.ts

```diff
@@ -0,0 +1,69 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { web3Accounts, web3Enable } from '@polkadot/extension-dapp';
+import { ISubstrateWallet } from '../../interfaces';
+
+class SubstrateWallet implements ISubstrateWallet {
+  substrateAccount?: string;
+  apiPromise: ApiPromise;
+
+  constructor(apiPromise: ApiPromise) {
+    this.apiPromise = apiPromise;
+  }
+
+  /**
+   * @name connectFromWssProvider
+   * @param wssProvider
+   * @returns {Promise<SubstrateWallet>}
+   * @description Initializes the SubstrateWallet from a wssProvider
+   */
+  static async connectFromWssProvider(
+    wssProvider: string
+  ): Promise<SubstrateWallet> {
+    const wsProvider = await SubstrateWallet.connectToApi(wssProvider);
```

Comment:
this method isn't async, right?

Reply:
🤔 I wonder why I added the await there... or perhaps I forgot to correct copilot